packages/renderer/src/lib/toast/ToastCustomUi.svelte

```diff
@@ -0,0 +1,65 @@
+<script lang="ts">
+import { faCheckCircle, faCircleExclamation, faSpinner } from '@fortawesome/free-solid-svg-icons';
+import { CloseButton, Link } from '@podman-desktop/ui-svelte';
+import { toast } from '@zerodevx/svelte-toast';
+import Fa from 'svelte-fa';
+
+import type { TaskInfo } from '/@api/taskInfo';
+
+let { toastId, taskInfo, onpop = () => {} }: { toastId: number; taskInfo: TaskInfo; onpop?: () => void } = $props();
+
+const closeAction = (): void => {
+  toast.pop(toastId);
+  onpop();
+};
+
+const executeAction = (): void => {
+  window.executeTask(taskInfo.id);
+};
+</script>
+
+<div
+  class="flex min-h-10 max-h-30 flex-col p-2 border-[var(--pd-button-tab-border-selected)] border rounded-md bg-[var(--pd-modal-bg)]"
+>
+  <div class="mb-1 flex flex-row items-center">
+    <div
+      class="mr-1 text-purple-500"
+      role="status"
+      aria-label={taskInfo.status}
+    >
+      {#if taskInfo.status === 'in-progress'}
+        <Fa class="animate-spin" icon={faSpinner} />
+      {:else if taskInfo.status === 'success'}
+        <Fa icon={faCheckCircle} />
+      {:else if taskInfo.status === 'failure'}
+        <Fa icon={faCircleExclamation} class="text-[var(--pd-state-error)]" />
+      {/if}
+    </div>
+
+    <div class="font-bold text-ellipsis line-clamp-1 max-w-32">
```

Comment:
Task name is using ellipsis at just past half-width, even when there is no percentage. Remove `max-w-32` here. Might need a grow, or conversely also a max-w on the whole card.